library/network/src/modules/SuSEFirewallServices.rb

```diff
@@ -752,6 +752,24 @@ def GetPossiblyConflictServices
   end
 
   class SuSEFirewalldServicesClass < SuSEFirewallServicesClass
+    SERVICES_DIR = ["/etc/firewalld/services", "/usr/lib/firewalld/services"]
```

Comment:
But SF2 uses _SERVICES_DIR_. The only other place that's being used is here (currently locally in my tree. I am still fixing stuff)

```rb
log.info "Reading FirewallD services from #{SERVICES_DIR.join(" and ")}"
```

We don't quite examine these directories ourselves since we use the FirewallD API to search for services. I am happy to drop this variable and the associated test.